tests/e2e/test_launch.py

```
@@ -0,0 +1,111 @@
+        if python_executable is None:
+            self.python_executable = os.path.join(
+                self.project_root, ".venv", "bin", "python"
+            )
+        else:
+            self.python_executable = python_executable
+
+        self.full_app_path = os.path.join(self.project_root, self.app_path)
+
+    def test_package_import(self):
+        """Test that the main module imports without missing dependencies."""
+        result = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)